wasm/src/main/java/com/dylibso/chicory/wasm/types/ExternalType.java

```diff
@@ -0,0 +1,43 @@
+package com.dylibso.chicory.wasm.types;
+
+import java.util.List;
+
+/**
+ * The type of external definition, import, or export.
+ * <p>
+ * See <a href="https://webassembly.github.io/spec/core/syntax/types.html#external-types">External Types</a> for
+ * reference.
+ */
+public enum ExternalType {
+    // note: keep in order
+    FUNCTION(0x00),
+    TABLE(0x01),
+    MEMORY(0x02),
+    GLOBAL(0x03),
+    TAG(0x04),
```

Comment:
What's a tag?
The spec mentions only 4 types:
https://webassembly.github.io/spec/core/syntax/types.html#external-types

func/table/mem/global
Do you have a reference?

Reply:
Sure. https://github.com/WebAssembly/exception-handling/blob/main/proposals/exception-handling/Exceptions.md#external_kind

Reply:
Thanks! Do you mind to add a comment with the link, otherwise LGTM